backend/services/deployments/tests/tests/test_api_internal.py

```diff
@@ -33,13 +31,14 @@
 )
 from client import SimpleArtifactsClient, ArtifactsClientError
 
+from client import internal_v1_client
+from internal_v1.rest import ApiException
+
 
 class TestInternalApiTenantCreate:
     def test_create_ok(self, api_client_int, clean_db):
```

Comment:
Unused fixture (also removed in enterprise). Applies to the rest of the tests in this class too.
```suggestion
    def test_create_ok(self, clean_db):
```

Reply:
this fixture actually is still there. I ported it to the oa3 client (left the requests though for the followup).